src/github.com/stellar/horizon/resource/account.go

```diff
@@ -71,3 +72,16 @@ func (this *Account) Populate(
 	this.Links.Data.PopulateTemplated()
 	return
 }
+
+// GetData returns decoded value for a given key. If the key does
+// not exist, empty slice will be returned.
+func (this *Account) GetData(key string) []byte {
+	data, exists := this.Data[key]
+	if !exists {
+		return nil
+	}
+
+	// We assume that horizon always returns valid base64 encoded string
+	decoded, _ := base64.StdEncoding.DecodeString(data)
```

Comment:
Please don't ignore errors.  IMO there are very few situations where this is acceptable.  There's nothing preventing this data from being corrupted for some reason, and we want to allow the detection of that failure.  Let the consumer of this method ignore the error if they so choose, but we should not.

A corrupt datafield in a horizon database should be a "holy shit! call the admin" moment... IMO the account endpoint should be returning a 500 error and reporting a bugreport to sentry.